README.md

```diff
@@ -76,6 +77,6 @@ If you want to hack on the extension itself, you should clone this repository, i
 
 ## ROADMAP
 
-* [ ] run `opa fmt` on save to reformat file
+* [x] run `opa fmt` on save to reformat file
```

Comment:
Nit: Just remove this from the list and add a bullet at the top to indicate it's supported.

Reply:
done!